versions/1/changelog.yml

```diff
@@ -4,6 +4,9 @@
 ##
 - version: 1.14.1-next
   changes:
+  - description: Add support for icons in dark mode
```

Comment:
Your PR should go to the bottom of the `changes` section.

Reply:
👍  Re-ordered